src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -155,11 +158,14 @@ private PSObject GetAppxObject(string packageName)
         private IReadOnlyList<string> GetVCLibsDependencies()
         {
             var vcLibsDependencies = new List<string>();
-            var vcLibsPackageObjs = this.psCmdlet.InvokeCommand
-                .InvokeScript(string.Format(GetAppxPackageByVersionCommand, VCLibsUWPDesktop, VCLibsUWPDesktopVersion));
+
+            var cmd = string.Format(GetAppxPackageByVersionCommand, VCLibsUWPDesktop, VCLibsUWPDesktopVersion);
+            this.psCmdlet.WriteDebug($"Running command: '{cmd}'");
+            var vcLibsPackageObjs = this.psCmdlet.InvokeCommand.InvokeScript(cmd);

```

Comment:
Invoking scripts built by strings always makes me a bit nervous. Are there checks in place to be sure no malicious commands can be injected into the string?

Reply:
> Invoking scripts built by strings always makes me a bit nervous

It's also against internal security recommendations to use scripts built by strings. I think they suggest you use ScriptBlocks

Reply:
Even though these are all constant strings, I agree that always using a ScriptBlock is the right thing to do to prevent any accidental injection attack possibilities.  It seems like it would be fairly straightforward to make wrapper functions for anything like this that contains the ScriptBlock and parameter addition to whatever `Invoke` function is for a ScriptBlock.

Reply:
The main reason I added InvokeScript is to avoid using the PowerShell obj and have to import Appx for PowerShell 7 every time. If there's no explicit import any cmdlet that uses this class will fail for pre 10.0.22453.0 builds. So I'm going back to use PowerShell, import Appx before any cmdlet in 7 and hide the warnings